### PR TITLE
Added navigator API's for browser support

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/tab-device-permissions/nodejs/src/components/CaptureImageDesktop.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    // var imageCapture: ImageCapture;
+
+    // Method to validate before capturing media
+    function captureMedia() {
+        // Method to ask for image capture permission and then select media
+
+        navigator.permissions.query({ name: 'camera' }).then(function (result) {
+            if (result.state == 'denied') {
+                alert("failed");
```
Comment (AbhijeetThakur-msft) [inline samples/tab-device-permissions/nodejs/src/components/CaptureImageDesktop.tsx:30]: are we not getting the image here? why do we require video part again?

**Review by AbhijeetThakur-msft — CHANGES_REQUESTED** [review]

File: samples/tab-device-permissions/nodejs/package.json [hunk 2]
```diff
@@ -39,5 +39,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/w3c-image-capture": "^1.0.5"
```
[the remark this answers is not in the document]
Comment (Prithvi-MSFT) [reply samples/tab-device-permissions/nodejs/package.json:44]: We are using ImageCapture class whose type definitions are not available by default in typescript. This package provides all those type definitions.

File: samples/tab-device-permissions/nodejs/src/components/CaptureImageDesktop.tsx (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/tab-device-permissions/nodejs/src/components/CaptureImageDesktop.tsx:30]: We are taking a snapshot of the video for capturing image as there is no separate navigator method for capturing image. So video part is required here.

**Review by AbhijeetThakur-msft — APPROVED** [review]

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, I am approving it.

File: samples/tab-device-permissions/nodejs/src/components/Tab.tsx [hunk 3]
```diff
@@ -6,8 +6,11 @@ import { Grid, Segment } from '@fluentui/react-northstar'
 import ScanBarCode from './ScanBarCode';
 import GetGeoLocation from './GetGeoLocation';
 import CaptureImage from './CaptureImage';
+import CaptureImageDesktop from './CaptureImageDesktop';
 import PeoplePicker from './PeoplePicker';
 import CaptureAudio from './CaptureAudio';
+import CaptureAudioDesktop from './CaptureAudioDesktop';
+import CaptureVideoDesktop from './CaptureVideoDesktop';
 import GetLocationDesktop from './GetLocationDesktop';
```
Comment (Wajeed-msft) [inline samples/tab-device-permissions/nodejs/src/components/Tab.tsx:14]: Could you please also include Notifications? 
navigator.permissions.query({ name: 'notifications' });

File: samples/tab-device-permissions/nodejs/src/components/Tab.tsx [hunk 4]
```diff
@@ -59,10 +62,24 @@ const Tab = () => {
...
+          /* Component to capture audio in browser */
+          content={<CaptureAudioDesktop />}
+        />
+        <Segment
+          /* Component to capture video in browser */
+          content={<CaptureVideoDesktop />}
+        />
+        <Segment
+          /* Component to Get/Show geo-Location in browser */
           content={<GetLocationDesktop />}
```
Comment (Wajeed-msft) [inline samples/tab-device-permissions/nodejs/src/components/Tab.tsx:80]: Screenshot change in ReadMe.md doesn't contain the 

image

Description still talks about how this is handled in browser.
image

Could you please capture the screenshot for browser?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]
Could you please also deploy this sample and include deployment script?

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**